src/EoS/Nuclear/NuclearEoS.h

```diff
@@ -21,20 +21,24 @@
 #define NUC_AUX_MEV2KELVIN    7     // AuxArray_Flt: convert MeV        to kelvin
 
 #define NUC_AUX_NRHO          0     // AuxArray_Int: nrho
-#define NUC_AUX_NEPS          1     // AuxArray_Int: neps
+#define NUC_AUX_NTORE         1     // AuxArray_Int: ntemp/neps
 #define NUC_AUX_NYE           2     // AuxArray_Int: nye
-#define NUC_AUX_NMODE         3     // AuxArray_Int: nmode
+#define NUC_AUX_NRHO_MODE     3     // AuxArray_Int: nrho_mode
+#define NUC_AUX_NMODE         4     // AuxArray_Int: nmode
+#define NUC_AUX_NYE_MODE      5     // AuxArray_Int: nye_mode
 
 
 // table indices
 #define NUC_TAB_ALL           0     // alltables
 #define NUC_TAB_ALL_MODE      1     // alltables_mode
 #define NUC_TAB_RHO           2     // logrho
-#define NUC_TAB_EPS           3     // logeps
+#define NUC_TAB_TORE          3     // logtemp/logenergy
 #define NUC_TAB_YE            4     // yes
-#define NUC_TAB_TEMP_MODE     5     // logtemp_mode
-#define NUC_TAB_ENTR_MODE     6     // entr_mode
-#define NUC_TAB_PRES_MODE     7     // logprss_mode
+#define NUC_TAB_RHO_MODE      5     // logrho_mode 
+#define NUC_TAB_ENTE_MODE     6     // logenergy_mode/logtemp_mode
```

Comment:
Would it be better to use `NUC_TAB_TORE_MODE` instead of `NUC_TAB_ENTE_MODE`

Reply:
Yes. Maybe `NUC_TAB_EORT_MODE` is better since T-based drvier uses `NUC_TAB_ENGY_MODE` and e-based drive uses `NUC_TAB_TEMP_MODE`.

Reply:
I suggest "TORE", instead of "EORT" (and "EN TE"), because other related variables use "TorE" (e.g., `NUC_TAB_TORE`)